spec/unit/provider/package/cab_spec.rb

```diff
@@ -18,7 +18,7 @@
 
 require "spec_helper"
 
-describe Chef::Provider::Package::Cab do
+describe Chef::Provider::Package::Cab, :windows_only do
```

Comment:
You might consider using platform agnostic paths in the tests like: `"#{ENV['TEMP']}/Test6.1-KB2664825-v3-x64.cab"` instead of `C:\\Temp\\Test6.1-KB2664825-v3-x64.cab`.